app/Jobs/EncodeTrackFile.php

```diff
@@ -50,20 +50,25 @@ class EncodeTrackFile extends Job implements SelfHandling, ShouldQueue
     /**
      * @var bool
      */
+    protected $autoPublishWhenComplete;
+    /**
+     * @var bool
+     */
     protected $isForUpload;
     /**
      * @var bool
      */
-    protected $autoPublishWhenComplete;
+    protected $isReplacingTrack;
 
     /**
      * Create a new job instance.
      * @param TrackFile $trackFile
      * @param bool $isExpirable
-     * @param bool $isForUpload indicates whether this encode job is for an upload
      * @param bool $autoPublish
+     * @param bool $isForUpload indicates whether this encode job is for an upload
+     * @param bool $isReplacingTrack
      */
-    public function __construct(TrackFile $trackFile, $isExpirable, $isForUpload = false, $autoPublish = false)
+    public function __construct(TrackFile $trackFile, $isExpirable, $autoPublish = false, $isForUpload = false, $isReplacingTrack = false)
```

Comment:
Include a type declaration for any new parameters you add to a function, and add them for existing parameters if you're familiar with them as well.